### DISCUSSION TITLE
How to get tmux/mc work properly with mouse clicks?

### DISCUSSION THREAD

**Discussion by gerroon** [Eugeny/tabby#9285] [Q&A]
HI

I use Cygwin to connect to my Linux boxes, I have tmux and mc on them. They normally except mouse clicks to pick pabes, windows etc. However none of that works when I use Cygwin shell to connect to remotes in Tabby.

Is there a way to fix it?

**Comment by marcemv90** [comment]
Hi 👋 ,

I'm in the same situation, the same happens with Git Bash. I raised the issue with more information, in case you want to have a look or add some comments:

https://github.com/Eugeny/tabby/issues/9783

Hope there's a fix.

Regards!

**Comment by gerroon** [comment]
I just tested 1.0.215 and the issue is still there :(  None of these areas are clickable, it works like text selection instead.


![image](https://github.com/user-attachments/assets/d50b0ac8-b288-4c50-8267-21d6d2a7839e)
